libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -209,10 +225,10 @@ export interface IExtractorOptions {
   includeNpmIgnoreFiles?: boolean;
 
   /**
-   * The folder where the PNPM "node_modules" folder is located. This is used to resolve packages linked
-   * to the PNPM virtual store.
+   * The subspace list to will be performed while extracting
+   *
    */
-  pnpmInstallFolder?: string;
+  subspaces?: IExtractorSubspace[];
```

Comment:
NOTE: It's a breaking change to remove "pnpmInstallFolder" and entirely replace it with "subspaces". If we do so, it's a **major** version bump.

Reply:
Agreed, to avoid breaking API compatibility we could allow both alternatives:  Either you specify `IExtractorState.transformPackageJson` etc, or else you can specify `IExtractorState.subspaces` (but not both at the same time).

Reply:
OK, I will rethink how can we make API more compatible here.